src/Features/Core/Portable/Common/GlyphExtensions.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.Completion;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class GlyphExtensions
```

Comment:
Does this mean we can delete src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs? It seems like it's doing the same thing.

Reply:
the same thing as what?

Reply:
OK, done. This should be better now. I also removed the LINQ code, and made `GetAccessibility(...)` O(n) rather than O(4n).